views/partials/_search-templates.slim

```diff
@@ -35,7 +35,6 @@ script id="search_result" type="text/html"
       div.action-wrapper
         a href="{{ site_link }}" Site
         a href="{{ docs_link }}" Docs
-        a href="{{ spec_link }}" Spec
```

Comment:
We'd have to re-implement the podspec she path in JS for this - doesn't make sense.